src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java

```diff
@@ -35,7 +35,7 @@ public static boolean isDateWithinLast3Months(LocalDate compareToDate, String qu
     public static boolean isCeasedDateAfterStartDate(LocalDate ceasedDate, LocalDate startDate, String qualifiedFieldName, Errors errors, String loggingContext) {
         if (ceasedDate.isBefore(startDate) || ceasedDate.isEqual(startDate)) {
             setErrorMsgToLocation(errors, qualifiedFieldName, ValidationMessages.CEASED_DATE_BEFORE_START_DATE_ERROR_MESSAGE.replace("%s", qualifiedFieldName));
-            ApiLogger.infoContext(loggingContext, qualifiedFieldName + " ceased date should be after start date");
+            ApiLogger.infoContext(loggingContext, qualifiedFieldName + " must be after the appointed date");
```

Comment:
use the ValidationMessages.CEASED_DATE_BEFORE_START_DATE_ERROR_MESSAGE variable here?

Reply:
Fixed